INSTALL.md

```diff
@@ -2,95 +2,38 @@
-because the next step is to build the documentations locally. 
-
-We suggest to install it into a `virtualenv` and for doing this we first need an environment.
-Let's check if you have correctly installed `virtualenv` by running:
-
-```bash
-which virtualenv
-```
-
-If yes, you will read its path location on the terminal. If not, you can install it following
-the [official instructions](https://virtualenv.pypa.io/en/latest/installation.html).
-
-Now we create an environment that we will use for building the docs using the following command:
-
-```bash
-virtualenv my-sphinx-env
-```
-
-Let's activate the new environment:
-
-```bash
-source my-sphinx-env/bin/activate
-```
-
-Now you will read the name of your environment `(my-sphinx-env)` on the right side of the command line.
-Here we are, now we can proceed with the installation of `sphinx`.
-
-You can do this running following the 
-[official `sphinx` instructions](https://smobsc.readthedocs.io/en/stable/usage/installation.html).
-On Linux or MacOS run:
-
-```bash
-pip install -U sphinx
-```
-
-or on the Windows prompt:
-
-```bash
-pip install -U --pre sphinx
-```
-
-Now the environment is ready to build the documentations.
-
-#### Building the docs
-
-The Qibo site needs the documentations to be generated locally. 
-
-###### Qibo docs
-
-For building `qibo`'s documentation run:
+#### We use `yarn` as dependence
 
-```bash
-cd public/packages/qibo/doc
-make html
-```
+You can install it following the
+[official instructions](https://yarnpkg.com/getting-started/install).
+With the following lines we suggest an installation procedure.
 
-If all is right, you must now have a folder called `build`, into `qibo/doc`.
+##### Installing `node` and `npm`
 
-###### Qibolab docs
+We developed the site using `yarn` as dependence.
 
-For building `qibolab`'s documentation come back to root into `qibogang.github.io` and run:
+In order to build this site on this way the first to do is to install `nodejs` and
+`npm`. Before doing this check if you have them already installed on your device.
 
-```bash
-cd public/packages/qibolab/doc
-make html
-```
+ ```bash
+node -v
+npm -v
+ ```
 
-If all is right, you must now have a folder called `build`, into `qibolab/doc`.
+If not, one possibility is to install it using the Advanced Packaging Tool `apt` 
+if you use an Ubuntu distribution. We suggest to follow the official instructions 
+[here](https://nodejs.dev/en/download/). If you need to add the PPA of `nodejs` 
+to your system you can use `curl` and select the desider node version 
+[here](https://github.com/nodesource/distributions#deb) and then proceed with `apt`.
 
-###### Qibocal docs
+##### Installing `yarn`
 
-For building `qibocal`'s documentation come back to root into `qibogang.github.io` and run:
+At this point, we suggest to install `yarn` globally:
 
 ```bash
-cd public/packages/qibocal/doc
-make html
+sudo npm install -g yarn
```

Comment:
Do not write explicitly instructions, since people might prefer alternative installation methods.
Reference the official page:
https://yarnpkg.com/getting-started/install